app/models/shelter.rb

```diff
@@ -4,6 +4,7 @@ class Shelter < ApplicationRecord
   validates :city, presence: true
 
   has_many :pets, dependent: :destroy
+  has_many :applications, through: :pets
```

Comment:
Great work adding this additional association! You can see when you do `Shelter.joins(:applications)` that SQL is under the hood, using the joins table and pets table to get to this one, but this AR association adds a nice shortcut to make the syntax much easier!